### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by muskan2532150**
- Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
- Use class based components (All components should be class-based)

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,52 +1,57 @@
-import { useState } from 'react';
+import React, { Component } from 'react';
 import calculate from './logic/Calculate';
```
Comment (mateo951) [inline src/components/Calculator.js:3]: There is a `Calculator.js` file inside the components directory ✔

File: src/components/Calculator.js [hunk 2]
```diff
@@ -1,52 +1,57 @@
...
 
-function Calculator() {
-  const [state, setstate] = useState({ total: '0', next: null, operation: null });
+export default class Calculator extends Component {
```
Comment (mateo951) [inline src/components/Calculator.js:4]: The `Calculator.js` component is class-based 🔥

File: src/components/Calculator.js [hunk 3]
```diff
@@ -1,52 +1,57 @@
-import { useState } from 'react';
```
Comment (mateo951) [inline src/components/Calculator.js:1]: Please be sure that your the `App.js` component is class-based as well because its part of the requirements 🤓

**Review by mateo951 — CHANGES_REQUESTED** [review]
### Hi @muskan2532150, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Review by aimemalaika — APPROVED** [review]
Hi @muskan2532150 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉

- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**